### PR TITLE
Last few checks for nutraceuticals

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

A few final checks to build on #824.
- Check EOL white space
- Check comments
- Check double spaces in-line
- Check all abbreviations defined at first use and used at least 3 times (note: left the regulatory agencies and their decrees defined since these seem worth defining as abbreviations regardless of how many times we invoke them)
- Check n-3 PUFA/PUFAs is used consistently (went with PUFAs?)
- Abstract/importance was too long so proposing edits :( I couldn't get the abstract below 270 for a long time but now it's actually about 200, so the text I proposed actually gives you 50 words to add back if desired! Importance is still 8 words over but I'm assuming that's not a huge deal.

Notes: 
- We can strike "Authors with similar contributions are ordered alphabetically" from the word doc we submit, since there are no authors with similar contributions on this one!
- Something odd is happening where some normal-looking line breaks between sentences in the same paragraph are being interpreted as a non-paragraph-style line break in the docx build. My guess is that the whitespace character is different on these lines (supported by the diff). We'll see if my hypothesis is correct when this builds -- I went through and deleted them in vim on a mac!

Because I read through this the other day and Ronan read through it yesterday, I just did a skim! I think everything is in pretty good shape so not super worried if a tense or two is off!

### Suggested reviewers (optional)

@RLordan @cgreene 
@agitter letting you know about the EOL whitespace thing -- looking at the diff, it does seem like the EOL character was different on these lines, but I am not finding how to get vim to show an ascii representation of whitespace characters without knowing a priori what whitespace character we're looking for... I'm not sure if this is a one-off or something to keep in mind for the future!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
> We can strike "Authors with similar contributions are ordered alphabetically" from the word doc we submit, since there are no authors with similar contributions on this one!

Do you want me to remove this from the front matter?  It's pretty easy to check if this is the nutraceuticals manuscript and leave the statement out if it is.  The only potential downside would be if more authors are added later and some have similar contributions.

The whitespace looks good in the PDF build.  Your fixes must have worked.  It's good to know about this for the future.

File: content/21.nutraceuticals.md [hunk 1]
```diff
@@ -178,18 +174,18 @@ Despite these findings, an RCT of 167 patients known as CITRUS ALI failed to sho
...
+However, the majority of supplementation studies investigate the intravenous infusion of vitamin C in severe patients.
 Therefore, there is a lack of studies investigating the potential prophylactic administration of vitamin C via oral supplementation for healthy individuals or potentially asymptomatic SARS-CoV-2 positive patients.
 Once again, vitamin C intake is part of a healthy diet and the vitamin likely presents minimal risk, but its potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.
-To maintain vitamin C status, it would be prudent for individuals to ensure that they consume the recommended dietary allowance (RDA) of vitamin C to maintain a healthy immune system [@doi:10.3390/nu12051466]. 
-The RDA recommended by the FDA is 75-90 mg/d, and EFSA recommends 110 mg/d [@doi:10.17226/9810].
+To maintain vitamin C status, it would be prudent for individuals to ensure that they consume the recommended dietary allowance of vitamin C to maintain a healthy immune system [@doi:10.3390/nu12051466].
+The recommended dietary allowance recommended by the FDA is 75-90 mg/d, and EFSA recommends 110 mg/d [@doi:10.17226/9810].
```
Comment (RLordan) [inline content/21.nutraceuticals.md:188]:
```suggestion
The recommended dietary allowance according to the FDA is 75-90 mg/d, whereas EFSA recommends 110 mg/d [@doi:10.17226/9810].
```

Comment (RLordan) [reply content/21.nutraceuticals.md:188]: lots of recommendations in one line haha

**Review by RLordan — APPROVED** [review]
All good with me @rando2, 1 suggestion

**Comment by RLordan** [comment]
PUFAs / PUFA used interchangeably, but it looks better to be consistent 👍

**Comment by rando2** [comment]
Wonderful!! @cgreene I will slack you the docx from this build (with the line about similar authors deleted), if that sounds good to @RLordan!

**Pull request merged**